### ISSUE TITLE
I found some errors on the Bangla cheat sheets!

### ISSUE THREAD

**Issue by Diptenusarkar** [github/training-kit#923]
## Overview
**TL;DR**
<One to two sentence description of the issue you are encountering or trying to solve.>

### Extra Detail
I found some grammatical errors on the Bangla page. so is that possible for you to assign this to me? so I can fix that.

### How to Contribute
- [x] Comment on this issue if you'd like to work/collaborate on it!
- [x] Fork the [repository](https://github.com/github/training-kit).
- [x] Push your changes to your repository.
- [ ] Submit a Pull Request
  - Base Dropdown: github/training-kit
  - Compare Dropdown: Your fork

#### Testing Locally
If you'd like to make and test changes locally (and see how they would look if merged), do the following:
- In your command line:
  - [Install Git](https://git-scm.com/)
  - [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
  - [ ] Run `script/setup`
  - [ ] Run `script/server`.
    - When successful, the script will initiate a local server at `http://127.0.0.1:4000/`.

### Questions?
- Leave a comment on this issue! Make sure to use @bruce  if you want a specific person's attention!
